src/SFA.DAS.Notifications.Application/Commands/SendSms/SendSmsCommandHandler.cs

```diff
@@ -44,6 +56,22 @@ protected override async Task HandleCore(SendSmsCommand command)
 
             Validate(command);
 
+            if (!Guid.TryParse(command.TemplateId, out Guid templateIdGuid))
```

Comment:
This check seems unnecessary, your command property is of type string. If it is ambiguous then you can change the name of the property to get rid of ambiguity. IMHO if you are not expecting a guid, it shouldn't even allow it.